spec/fluent/plugin/shared.rb

```diff
@@ -289,7 +289,12 @@
 
 shared_examples_for 'instruments record' do
   before do
-    driver.run(default_tag: tag) { driver.feed(event_time, message) }
+    # Should not shut down driver because it will be used in subsequent tests.
+    driver.run(default_tag: tag, shutdown: false) { driver.feed(event_time, message) }
```

Comment:
The driver is also used in the following tests.

https://github.com/fluent/fluent-plugin-prometheus/blob/f8e841863f89dfb38878a53845d314814838778f/spec/fluent/plugin/shared.rb#L334-L337